### ISSUE TITLE
Error on lec2-4-1

### ISSUE THREAD

**Issue by HirokiNakahara** [HirokiNakahara/MachineLearning_Lecture#3]
Google colabで実行中にエラー「Import error cannot import name "Imputer" from sklearn.preprocessing」が発生しました。
対処策としては
「from sklearn.preprocessing import Imputer」
を
「from sklearn.impute import SimpleImputer」
に書き換えるとうまくいきました。

同様に
「imr = Imputer(missing_values='NaN', strategy='mean', axis=0)」
を
「imr = SimpleImputer(missing_values=np.nan, strategy='mean')」
に修正する必要があります。

反映をお願いします。
